src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -1309,7 +1309,12 @@ private async Task ValidateUnusedIdsAsync(HashSet<string> unusedIds, string data
 
         public static unsafe bool IsBase64String(string base64)
         {
-            int base64Size = (int)Math.Ceiling((double)base64.Length / 3) * 4;
+            // base64string length has to be a product of 4
+            var paddingLen = base64.Length%4;
+            for (int i = 0; i < paddingLen; i++)
+                base64 += "=";
```

Comment:
You are now allocating, why? The whole point is to do this wihtout allocations

Reply:
Looks like DatabaseIds are not Base64 strings so the whole code can be removed, I do not see a point for having it